daytrader-accountsapp/daytrader-entities/src/test/java/org/apache/geronimo/daytrader/javaee6/entities/AccountDataBeanTest.java

```diff
@@ -0,0 +1,147 @@
+import static org.powermock.api.support.membermodification.MemberMatcher.method;
+
+@RunWith(PowerMockRunner.class)
+public class AccountDataBeanTest {
+
+  @PrepareForTest({AccountDataBean.class, TradeConfig.class})
+  @Test
+  public void testGetRandomInstance() throws Exception {
+    PowerMockito.mockStatic(TradeConfig.class);
+
+    final Date creationDate = new Date(1555332012015L);
+    final Date lastLogin = new Date(1555332612015L);
+    PowerMockito.whenNew(Date.class).withParameterTypes(long.class).withArguments(anyLong()).thenReturn(creationDate);
+    PowerMockito.whenNew(Date.class).withNoArguments().thenReturn(lastLogin);
+
+    ((PowerMockitoStubber)PowerMockito.doReturn(new BigDecimal(0.1f)).doReturn(new BigDecimal(0.2f)))
+    .when(TradeConfig.class, method(TradeConfig.class, "rndBigDecimal", float.class)).withArguments(anyFloat());
+
+    ((PowerMockitoStubber)PowerMockito.doReturn(0).doReturn(1).doReturn(2))
+    .when(TradeConfig.class, method(TradeConfig.class, "rndInt", int.class)).withArguments(anyInt());
+
+    PowerMockito.doReturn("foo").when(TradeConfig.class);
+    TradeConfig.rndUserID();
+
+    final AccountDataBean accountDataBean = AccountDataBean.getRandomInstance();
+    Assert.assertEquals("\n\tAccount Data for account: 0"
+                + "\n\t\t   loginCount:1"
+                + "\n\t\t  logoutCount:2"
+                + "\n\t\t    lastLogin:Mon Apr 15 13:50:12 BST 2019"
+                + "\n\t\t creationDate:Mon Apr 15 13:40:12 BST 2019"
+                + "\n\t\t      balance:0.100000001490116119384765625"
+                + "\n\t\t  openBalance:0.20000000298023223876953125"
+                + "\n\t\t    profileID:foo", accountDataBean.toString());
+  }
+
+  @Test
+  public void testHashCodeEmpty() {
+    final AccountDataBean accountDataBean = new AccountDataBean();
+    Assert.assertEquals(0, accountDataBean.hashCode());
+  }
+
+  @Test
+  public void testHashCode() {
+    final AccountDataBean accountDataBean = 
+      new AccountDataBean(0, 1, 2, new Date(1555332612015L), new Date(1555332012015L), new BigDecimal(0.1f), new BigDecimal(0.2f), "foo");
+    Assert.assertEquals(101574, accountDataBean.hashCode());
+  }
+
+  @Test
+  public void testPrintToString() throws Exception {
+    final AccountDataBean accountDataBean = 
+      new AccountDataBean(0, 1, 2, new Date(1555332612015L), new Date(1555332012015L), new BigDecimal(0.1f), new BigDecimal(0.2f), "foo");
+    Assert.assertEquals("\n\tAccount Data for account: 0"
+                + "\n\t\t   loginCount:1"
+                + "\n\t\t  logoutCount:2"
+                + "\n\t\t    lastLogin:Mon Apr 15 13:50:12 BST 2019"
+                + "\n\t\t creationDate:Mon Apr 15 13:40:12 BST 2019"
+                + "\n\t\t      balance:0.100000001490116119384765625"
+                + "\n\t\t  openBalance:0.20000000298023223876953125"
+                + "\n\t\t    profileID:foo", accountDataBean.toString());
+    accountDataBean.print();
```

Comment:
What is being tested about print?

Reply:
Nothing. But the method is covered at least... It basically outputs `toString` to the console if I recall correctly.

Reply:
Then add this info as a comment and say that no exception is expected to be thrown.

Reply:
@peterschrammel Done